### ISSUE TITLE
Add location function in Web App.

### ISSUE THREAD

**Issue by secondl1ght** [teambtcmap/btcmap.org#18]
This should feed the Noob bot.

**Comment by secondl1ght** [comment]
@dadofsambonzuki more info here please

**Comment by dadofsambonzuki** [comment]
The ability to add a location from within the map view, perhaps by placing an marker and adding the basic info as per the Noob form.

**Comment by secondl1ght** [comment]
do we really need that or could we just add a button that links to the /add-location page?

**Comment by dadofsambonzuki** [comment]
Agreed. No point in duplicating functionality. Link to /add-location is fine.

**Comment by secondl1ght** [comment]
Sounds good - I will add this
